database/migrations/2021_04_29_150026_create_follow_requests_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFollowRequestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('follow_requests', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('follow_id');
+            $table->timestamps();
+
+            $table->unique(['user_id', 'follow_id']);
+        });
```

Comment:
```suggestion
            $table->unique(['user_id', 'follow_id']);
            
            $table->foreign('user_id')
                      ->references('id')
                      ->on('users')
                      ->cascadeOnDelete();
            $table->foreign('follow_id')
                      ->references('id')
                      ->on('users')
                      ->cascadeOnDelete();
        });
```

Please test the migration before merging the PR. It's just typed in this shitty text-field.